Makefile

```diff
@@ -0,0 +1,211 @@
+	rm -rf $(MADARA_DATA_PATH) && \
+	cargo run --release -- --name madara --base-path $(MADARA_DATA_PATH) --rpc-port 9944 --rpc-cors "*" --rpc-external --sequencer --chain-config-path configs/presets/devnet.yaml --feeder-gateway-enable --gateway-enable --gateway-external --gas-price 0 --blob-gas-price 0 --rpc-methods unsafe --no-l1-sync
+
+core-contract:
+	cd $(BOOTSTRAPPER_PATH) && \
+	rm -f $(BOOTSTRAP_OUTPUT_PATH) && \
+	git checkout $(BOOTSTRAPPER_COMMIT) && \
+	RUST_LOG=debug cargo run --release -- --mode core --operator-address $(OPERATOR_ADDRESS) --output-file $(BOOTSTRAP_OUTPUT_PATH) --verifier-address $(VERIFIER_ADDRESS) && \
+	cat $(BOOTSTRAP_OUTPUT_PATH) && \
+    $(call save_json,"CORE_CONTRACT_ADDRESS",$$(jq -r .starknet_contract_address $(BOOTSTRAP_OUTPUT_PATH))) && \
+    $(call save_json,"CORE_CONTRACT_IMPLEMENTATION_ADDRESS",$$(jq -r .starknet_contract_implementation_address $(BOOTSTRAP_OUTPUT_PATH)))
+
+madara:
+	$(call update_core_contract_address) && \
+	cd $(MADARA_PATH) && \
+	git checkout $(MADARA_COMMIT) && \
+	cargo run --release -- --name madara --base-path $(MADARA_DATA_PATH) --rpc-port 9944 --rpc-cors "*" --rpc-external --sequencer --chain-config-path configs/presets/devnet.yaml --feeder-gateway-enable --gateway-enable --gateway-external --gas-price 2 --blob-gas-price 2 --strk-gas-price 2 --strk-blob-gas-price 2 --rpc-methods unsafe --l1-endpoint http://0.0.0.0:8545
+
+eth-bridge:
+	cd $(BOOTSTRAPPER_PATH) && \
+	git checkout $(BOOTSTRAPPER_COMMIT) && \
+	export CORE_CONTRACT_ADDRESS=$$(jq -r '.CORE_CONTRACT_ADDRESS' $(ENV_FILE)) && \
+	export CORE_CONTRACT_IMPLEMENTATION_ADDRESS=$$(jq -r '.CORE_CONTRACT_IMPLEMENTATION_ADDRESS' $(ENV_FILE)) && \
+	echo "TODO: set core contract address" && \
+	RUST_LOG=debug cargo run --release -- --mode eth-bridge --core-contract-address $$CORE_CONTRACT_ADDRESS --core-contract-implementation-address $$CORE_CONTRACT_IMPLEMENTATION_ADDRESS  --output-file $(BOOTSTRAP_OUTPUT_PATH) && \
+	$(call save_json,"L1_BRIDGE_ADDRESS","$$(jq -r .eth_bridge_setup_outputs.l1_bridge_address $(BOOTSTRAP_OUTPUT_PATH))") && \
+    $(call save_json,"L2_ETH_TOKEN_ADDRESS","$$(jq -r .eth_bridge_setup_outputs.l2_eth_proxy_address $(BOOTSTRAP_OUTPUT_PATH))") && \
+    $(call save_json,"L2_ETH_BRIDGE_ADDRESS","$$(jq -r .eth_bridge_setup_outputs.l2_eth_bridge_proxy_address $(BOOTSTRAP_OUTPUT_PATH))")
+
+udc:
+	cd $(BOOTSTRAPPER_PATH) && \
+	git checkout $(BOOTSTRAPPER_COMMIT) && \
+	RUST_LOG=debug cargo run --release -- --mode udc
+
+pathfinder:
+	cd $(PATHFINDER_PATH) && \
+	git checkout $(PATHFINDER_COMMIT) && \
+	rm -rf $(PATHFINDER_DATA_PATH) && \
+	cargo run --release --bin pathfinder -- --ethereum.url wss://eth-sepolia.g.alchemy.com/v2/WIUR5JUZXieEBkze6Xs3IOXWhsS840TX --chain-id MADARA_DEVNET --feeder-gateway-url http://localhost:8080/feeder_gateway --gateway-url  http://localhost:8080/gateway --network custom --storage.state-tries archive --data-directory $(PATHFINDER_DATA_PATH)
```

Comment:
Seems lie the pvt key is being exposed here : wss://eth-sepolia.g.alchemy.com/v2/WIUR5JUZXieEBkze6Xs3IOXWhsS840TX
Can we deprecate this one and set up a env based pick up for this url ?

Reply:
good catch, fixed